tests/test_series.py

```diff
@@ -86,6 +87,7 @@
         UIntDtypeArg,
         VoidDtypeArg,
     )
+    from pandas._typing import UnknownSeries  # noqa: F401
```

Comment:
I think you can move this up to the previous import in lines 63-73

Reply:
I think that, but then `ruff` removes it as it's unused (as it's only used in `check`, which `ruff` doesn't recognise as usage? 

I think it's similar to

https://github.com/pandas-dev/pandas-stubs/blob/fd6188aec2e823679bbdb9416c88ce9e3b9f988a/tests/test_series.py#L91

Reply:
If you do
```python
if TYPE_CHECKING:
    from pandas.core.series import (
        OffsetSeries,
        TimedeltaSeries,
        TimestampSeries,
        UnknownSeries,
    )
else:
    TimedeltaSeries: TypeAlias = pd.Series
    TimestampSeries: TypeAlias = pd.Series
    OffsetSeries: TypeAlias = pd.Series
    UnknownSeries: TypeAlias = pd.Series
```
Then take away the quotes from `UnknownSeries` in the test, ruff won't remove it.